beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -182,8 +173,7 @@ impl<E: EthSpec> PendingComponents<E> {
         kzg_verified_data_columns: I,
     ) -> Result<(), AvailabilityCheckError> {
         for data_column in kzg_verified_data_columns {
-            // TODO(das): Add equivalent checks for data columns if necessary
-            if !self.data_column_exists(data_column.index()) {
+            if self.get_cached_data_column(data_column.index()).is_none() {
```

Comment:
Checks not necessary, we check the inclusion proof both via RPC and gossip. Inlined the function as it makes it easier to follow what's actually doing the `data_column_exists` check